docs/porting/third-party-deps.md

```diff
@@ -0,0 +1,50 @@
+# Analyzing your 3rd Party Dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+
+## Prerequisites
+
+This article will assume you are using Windows and Visual Studio.  For your NuGet package dependencies, you may also wish to download the [Nuget Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).
+
+## Analyzing NuGet Packages
+
+Analyzing NuGet packages for portability is very easy.  Because a NuGet package is itself a set of folders which contain platform-specific assemblies, all you have to do is check to see if there is a folder which contains a .NET Core assembly.
+
+There are a few different folder names that are valid for .NET Core (these are also called Target Framework Monikers, or TFMs for short).  They are:
+
+1. `netstandard1x` or `netstandard1.x`, where `x` is a value from `0` to `5`.
+2. `dotnet5x` or `dotnet5.x`, where `x` is a value from `1` to `5`.
+3. `dnxcore50`
```

Comment:
These need to change with the final release. The trouble is that some of these will be valid, so you might want to add netcoreapp1.0 and explain dnxcore50 as an "old one" or something.